mobile/callog/status_request_log.go

```diff
@@ -145,11 +148,34 @@ func Log(logger *zap.Logger, method string, params any, resp any, startTime time
 	)
 }
 
+func LogSignal(logger *zap.Logger, eventType string, event interface{}) {
+	if logger == nil {
+		return
+	}
+	logger.Debug("signal",
+		zap.String("type", eventType),
+		dataField("event", event),
```

Comment:
Small detail @igor-sirotin in many circumstances I had to reduce the noise from signals because some events happen too frequently and also some events I didn't care about the payload. The nice thing in mobile and Clojure is that I can change the log level programmatically while the app runs without reloading anything. Perhaps if we eventually use log namespaces it would be a nice enough solution to allow the dev to manipulate the level of noise from signals.

Reply:
The API logger is a completely separate instance that does not recognize namespaces or log level settings. We can easily integrate it with the namespaces mechanism though. To achieve this, we can add a new field to `InitializeApplication` called `APILogNamespaces` and use it to initialize the filtering core.